### ISSUE TITLE
IPAllowList does not show allowed IPs in Dashboard as IPWhiteList does

### ISSUE THREAD

**Issue by schenklklopfer** [traefik/traefik#10434]
### Welcome!

- [X] Yes, I've searched similar issues on [GitHub](https://github.com/traefik/traefik/issues) and didn't find any.
- [X] Yes, I've searched similar issues on the [Traefik community forum](https://community.traefik.io) and didn't find any.

### What did you do?

Changed my middleware from `IPWhiteList` to `IPAllowList`.

Open Traefik Dashboard and have a look on the configured middleware.
(Or open a HTTP Router that uses the middleware)

The page shows only:
- TYPE: ipallowlist
- PROVIDER: File
- STATUS: Success

`IPWhiteList` used to show also the configured IP Adresses from "sourceRange".

When I change back to `IPWhiteList`, the list of configured IPs is shown.

### What did you see instead?

No configured IPs.

### What version of Traefik are you using?

Version:      2.11.0
Codename:     cheddar
Go version:   go1.22.0
Built:        2024-02-12T15:26:45Z
OS/Arch:      linux/amd64

### What is your environment & configuration?

Before and working:
```
http:
  middlewares:
    IP_allow:
      ipWhiteList:
```

After and not working:
```
http:
  middlewares:
    IP_allow:
      ipAllowList:
```

**Comment by rtribotte** [comment]
Hello @schenklklopfer,

Thanks for reporting this!

The vue component for the IPAllowList is indeed missing in `webui/src/components/_commons/PanelMiddlewares.vue`, we marked this issue as bug-confirmed accordingly.

**Comment by rtribotte** [comment]
Fixed by #10459
